### DISCUSSION TITLE
How to Remove Previous Years Contributions from Your GitHub Profile?

### DISCUSSION THREAD

**Discussion by Turmaxx** [community/community#61144] [Other Feature Feedback, Questions, & Ideas]
Hey, 

I recently opened an old project that had commits from 2019 and pushed it to my GitHub repository. However, when the project was pushed, my GitHub profile displayed that I had contributions in 2019 and 2020 which I did even though I only joined GitHub in 2021. Can you please guide me on how to remove these 2019 and 2020 from my profile? I have amended the commits to 2021, but these years are still showing up.

Thank you

**Comment by ZIDANIDROS** [comment]
Have you succeeded in deleting the years?

explain me if ,you succeeded

**Reply by Turmaxx** [reply]
I haven't found a solution for this instead I kept it as it.
